src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2860,6 +2860,23 @@ private nuint printFieldName(CORINFO_FIELD_STRUCT_* fld, byte* buffer, nuint buf
             return PrintFromUtf16(field.Name, buffer, bufferSize, requiredBufferSize);
         }
 
+        private uint getThreadLocalFieldInfo(CORINFO_FIELD_STRUCT_* fld)
+        {
+            // Implemented for JIT only for now.
+
+            // This is present so compiler doesn't complain about method not being static.
```

Comment:
We should do this instead

```csharp
#pragma warning disable CA1822 // Mark members as static
```

We already do this for some methods in this file. Maybe we should just put this pragma at the beginning of the file. These analyzers cause more annoyances than they're worth.

Reply:
just surrounded the method for now.